hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -210,10 +210,10 @@ private HapiSpec validateNewTokenAssociations() {
                         getTxnRecord(creationTxn)
                                 .hasPriority(recordWith()
                                         .autoAssociated(accountTokenPairs(List.of(
-                                                Pair.of(treasury, A_TOKEN),
                                                 Pair.of(fractionalCollector, A_TOKEN),
                                                 Pair.of(selfDenominatedFixedCollector, A_TOKEN),
-                                                Pair.of(otherSelfDenominatedFixedCollector, A_TOKEN))))),
+                                                Pair.of(otherSelfDenominatedFixedCollector, A_TOKEN),
+                                                Pair.of(treasury, A_TOKEN))))),
```

Comment:
What's the difference having this here as opposed to underneath `.autoAssociated(...)`?

Reply:
Since we sorted all auto-associations, it fails in assertions.